src/front-end/typescript/lib/pages/landing.tsx

```diff
@@ -50,30 +52,35 @@ const update: Update<State, Msg> = ({ state, msg }) => {
 
 const view: ComponentView<State, Msg> = ({state, dispatch}) => {
   return (
-    <Row>
-      <Col xs='12'>
-        Landing page coming soon.
-      </Col>
-      <Col xs='12' className='pt-7'>
-        <button onClick={() => dispatch(adt('showToast'))}>Show Toast</button>
-      </Col>
-      <Col xs='12' md='8' className='pt-7'>
-        <Accordion
-          toggle={() => dispatch(adt('toggleAccordion'))}
-          color='info'
-          title='Inception'
-          titleClassName='h3'
-          icon='map'
-          iconWidth={2}
-          iconHeight={2}
-          iconClassName='mr-3'
-          chevronWidth={1.5}
-          chevronHeight={1.5}
-          open={state.showAccordion}>
-          Children!
-        </Accordion>
-      </Col>
-    </Row>
+    <Container xs='12'>
+      <Row>
+        <Col xs={{ size: 6, offset: 3 }} className='text-center mt-3'>
+          <h1 style={{
+            lineHeight: '60px'
+          }}>
+            Discover Unique Opportunities to Collaborate with the BC Public Sector.
+          </h1>
+        </Col>
+      </Row>
+      <Row>
+        <Col xs={{ size: 6, offset: 3 }} className='text-center'>
+          The Digital Marketplace is a new platform that will help build an ecosystem of innovation and collaboration between tech entrepreneurs and BC's public sector.
+        </Col>
+      </Row>
+      <Row>
+        <Button className='mx-auto mt-6 mb-6' color='primary'><Icon hover name='search' /><span className='ml-2'>Browse Opportunities</span></Button>
+      </Row>
+      <Row>
+        <Col className='text-center'>
+          <span style={{fontSize: '24px', fontWeight: 'bolder', verticalAlign: 'middle'}}>{formatAmount(1346)}</span>
+          <span className='ml-2' style={{fontSize: '14px', color: '#6C757D', verticalAlign: 'middle'}}>Total Opportunities Awarded</span>
+        </Col>
+        <Col className='text-center'>
+          <span style={{fontSize: '24px', fontWeight: 'bolder', verticalAlign: 'middle'}}>{formatAmount(13782000, '$')}</span>
+          <span className='ml-2' style={{fontSize: '14px', lineHeight: '21px', color: '#6C757D', verticalAlign: 'middle'}}>Total Value of All Opportunities</span>
```

Comment:
`<span className='ml-2 font-size-small'>`

Avoid using `lineHeight` in favour of the margin/padding bootstrap utilities and flexbox as needed.

Also, avoid using hex codes in TypeScript! In this case, you can simply use the `text-secondary` class.